doc/tutorials/pre-launched-rt.rst

```diff
@@ -0,0 +1,115 @@
+such as Linux. Also, the platform must support booting ACRN with
+multiple kernel images. So, your platform must have:
+
+- Two hard disk drives, one for the Pre-Launched RT and one for the Service
+  VM
+- Two network devices
+- GRUB multiboot support
+
+Example of Pre-Launched RT
+**************************
+
+Take the Whiskey Lake WHL-IPC-I5 board (as described in :ref:`hardware`) for
+example. This platform can connect both an NVMe and a SATA drive and has
+two Ethernet ports. We will passthrough the SATA and Ethernet 03:00.0
+devices into the Pre-Launched RT VM, and give the rest of the devices to
+the Service VM.
+
+Install SOS with Grub on NVMe
+=============================
+
+As with the Hybrid and Logical Partition scenarios, the Pre-Launched RT
+mode must boot using GRUB.  The ACRN hypervisor is loaded as a GRUB
+multiboot kernel, while the Pre-Launched RT kernel and Service VM
+kernels are loaded as multiboot modules. The ACRN hypervisor, Service
+VM, and Pre-Launched RT kernel images are all located on the NVMe drive.
+We recommend installing Ubuntu on the NVMe drive as the Service VM OS,
+which also has the required GRUB image to launch Pre-Launched RT mode.
+Refer to :ref:`Run Ubuntu as the Service VM <Ubuntu Service OS>`, to
+install Ubuntu on the NVMe drive, and use grub to launch the Service VM.
+
+Install Pre-Launched RT Filesystem on SATA and Kernel Image on NVMe
+===================================================================
+
+The Pre-Launched Preempt RT Linux use Clearlinux as rootfs. Refer to
+:ref:`Burn the Preempt-RT VM image onto the SATA disk <install_rtvm>` to
+download the RTVM image and burn it to the SATA drive. The Kernel should
+be on the NVMe drive along with GRUB. You'll need to copy the RT kernel
+to the NVMe drive. Once you have successfully installed and booted
+Ubuntu from the NVMe drive, you'll then need to copy the RT kernel from
+the SATA to the NVMe drive:
+
+.. code-block:: none
+
+   # mount /dev/nvme0n1p1 /boot
+   # mount /dev/sda1 /mnt
+   # cp /mnt/bzImage /boot/EFI/BOOT/bzImage_RT
+
+Build ACRN with Pre-Launched RT Mode
+====================================
+
+The ACRN VM configuration framework can easily configure resources for
+Pre-Launched VMs. On Whiskey Lake WHL-IPC-I5, to passthrough SATA and
+ethernet 03:00.0 devices to the Pre-Launched RT VM, build ACRN with:
+
+.. code-block:: none
+
+   make BOARD_FILE=$PWD/misc/acrn-config/xmls/board-xmls/whl-ipc-i5.xml SCENARIO_FILE=$PWD/misc/acrn-config/xmls/config-xmls/whl-ipc-i5/hybrid_rt.xml
+
+You should replace ACRN with yours. After the build completes, these
+files are changed by the ACRN build tools:
+
+.. code-block:: none
+
+   modified:   hypervisor/arch/x86/configs/whl-ipc-i5.config
+   modified:   hypervisor/arch/x86/configs/whl-ipc-i5/board.c
+   modified:   hypervisor/arch/x86/configs/whl-ipc-i5/misc_cfg.h
+   modified:   hypervisor/arch/x86/configs/whl-ipc-i5/pci_devices.h
+
```

Comment:
@taoyuhong Those configure files location is changed recently, could you please update it?

Reply:
Hi, @jsun26intel that section is out of date, now has hybrid_rt scenario, and acrn build tools do not modify code . can I just remove this section?